sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/data/DummyTrackingConnector.java

```diff
@@ -0,0 +1,58 @@
+package life.qbic.portal.sampletracking.data;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Random;
+import life.qbic.portal.sampletracking.view.projects.viewmodel.ProjectStatus;
+
+public class DummyTrackingConnector implements SampleStatusProvider, ProjectStatusProvider {
```

Comment:
Why is there a Dummy class in src/main?

Reply:
Can be removed. I had to implement it to develop while systems were down.